frontend/public/static/image/selectcol-btn-sub.svg

```diff
@@ -0,0 +1,8 @@
+<svg width="14" height="12" viewBox="0 0 14 12" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We should migrate everything over to use one type of icon.

Reply:
This is a custom icon Avni made, since we couldn't find something that fit our needs in font-awesome. Though yes, I think eventually we should move to feather since font-awesome doesn't have all the icons we want